### PR TITLE
Update docu: Add visual path to release

### PR DESCRIPTION
**Pull request by leonardpahlke**
Signed-off-by: leonardpahlke <leonard.pahlke@googlemail.com>

This PR adds the diagrams that @annajung was working on last summer (see [A visual for the release team](https://mermaid-js.github.io/mermaid-live-editor/view/#eyJjb2RlIjoiZ3JhcGggTFJcbiAgICBTVEFSVFtTdGFydCBvZiBSZWxlYXNlIEN5Y2xlXVxuICAgIFNUQVJUIC0tPiBTRVRcbiAgICBTVEFSVCAtLT4gTU9OSVRPUlxuICAgIEVGIC0tPiBGQk9EXG4gICAgRUYgLS0-IENNVFxuICAgIEVJUCAtLT4gQ0ZcbiAgICBDRiAtLT4gRFBEXG4gICAgVEYgLS0-IEZEUk5cblxuICAgIEVORCgoUmVsZWFzZSBEYXkpKVxuICAgIFRFU1QgLS0-IEVORFxuICAgIEVBQVJCIC0tPiBFTkRcbiAgICBEQ0YgLS0-IEVORFxuICAgIEZCUkQgLS0-IEVORFxuICAgIFJOQyAtLT4gRU5EXG5cbiAgICBzdWJncmFwaCBFbmhhbmNlbWVudHNcbiAgICAgICAgU0VUW1N0YXJ0IEVuaGFuY2VtZW50cyBUcmFja2luZ10gLS0-IEVGW0VuaGFuY2VtZW50cyBGcmVlemVdXG4gICAgICAgIEVGIC0tPiBDRltDb2RlIEZyZWV6ZV1cbiAgICAgICAgQ0YgLS0-IFRGW1Rlc3QgRnJlZXplXVxuICAgIGVuZFxuXG5cbiAgICBzdWJncmFwaCBEb2NzXG4gICAgICAgIERQRFtEb2NzIFBsYWNlaG9sZGVyIERlYWRsaW5lXSAtLT4gRFJEW0RvY3MgUmVhZHkgRm9yIFJldmlldyBEZWFkbGluZV1cbiAgICAgICAgRFJEIC0tPiBEQ0ZbRG9jcyBDb21wbGV0ZSBEZWFkbGluZV1cbiAgICBlbmRcblxuXG4gICAgc3ViZ3JhcGggQ29tbXNcbiAgICAgICAgRkJPRFtGZWF0dXJlIEJsb2cgT3B0LWluIERlYWRsaW5lXSAtLT4gUkJSRFtSZWxlYXNlIEJsb2cgUmVhZHkgRm9yIFJldmlldyBEZWFkbGluZV1cbiAgICAgICAgUkJSRCAtLT4gRkJSRFtGZWF0dXJlIEJsb2dzIFJlYWR5IEZvciBSZXZpZXcgRGVhZGxpbmVdXG4gICAgZW5kXG5cblxuICAgIHN1YmdyYXBoIFJlbGVhc2UgTm90ZXNcbiAgICAgICAgQ01UW1N0YXJ0IENvbGxlY3RpbmcgTWFqb3IgVGhlbWVzXSAtLT4gRkRSTltTdGFydCBGaW5hbCBEcmFmdCBvZiBSZWxlYXNlIE5vdGVzXVxuICAgICAgICBGRFJOIC0tPiBSTkNbUmVsZWFzZSBOb3RlIENvbXBsZXRlXVxuICAgIGVuZFxuXG4gICAgc3ViZ3JhcGggQnVnIFRyaWFnZVxuICAgICAgICBUQVtUcmFjayBBbGwgSXNzdWVzIGFuZCBQUnNdIC0tPiBFSVBbRXNjYWxhdGUgSXNzdWVzIGFuZCBQUnNdXG4gICAgICAgIFRGIC0tPiBFQUFSQltFbnN1cmUgQXR0ZW50aW9uIG9uIEFueSBSZWxlYXNlLUJsb2NraW5nIGFuZCBNYWpvciBCdWcgRml4IFBSc11cbiAgICBlbmRcblxuICAgIHN1YmdyYXBoIENJIFNpZ25hbFxuICAgICAgICBNT05JVE9SW01vbml0b3IgRTJFIFRlc3RzIGFuZCBBbGwgSm9icyBJbiBTSUcgUmVsZWFzZSBEYXNoYm9hcmRzXVxuICAgICAgICBNT05JVE9SIC0tPiBPUEVOW09wZW4gSXNzdWVzIEZvciBGYWlsaW5nIE9yIEZsYWtpbmcgSm9ic11cbiAgICAgICAgQ0YgLS0-IEVTQ0FMQVRFW0VzY2FsYXRlIEFueSBOZXcgRmFpbHVyZXNdXG4gICAgICAgIE9QRU4gLS0-IEVTQ0FMQVRFXG4gICAgICAgIEVBQVJCIC0tPiBURVNUW0Vuc3VyZSBUZXN0cyBoYXZlIFN0YWJpbGl6ZWRdXG4gICAgICAgIEVTQ0FMQVRFIC0tPiBURVNUXG4gICAgZW5kIiwibWVybWFpZCI6IntcbiAgXCJ0aGVtZVwiOiBcImRlZmF1bHRcIlxufSIsInVwZGF0ZUVkaXRvciI6dHJ1ZSwiYXV0b1N5bmMiOnRydWUsInVwZGF0ZURpYWdyYW0iOnRydWV9) & [A visual for the KEP author](https://mermaid-js.github.io/mermaid-live-editor/view/#eyJjb2RlIjoiZ3JhcGggVERcbiAgICBLRVBfQVVUSE9SW0tFUCBBVVRIT1JdIC0tPiBESVNDVVNTXG5cbiAgICBzdWJncmFwaCBSZWxlYXNlIFRlYW0gU2NoZWR1bGVcbiAgICAgICAgU1RBUlRbU3RhcnQgb2YgUmVsZWFzZSBDeWNsZV1cbiAgICAgICAgU1RBUlRfRU5IQU5DRU1FTlRTX1RSQUNLSU5HW1N0YXJ0IEVuaGFuY2VtZW50cyBUcmFja2luZ11cbiAgICAgICAgRU5IQU5DRU1FTlRTX0ZSRUVaRVtFbmhhbmNlbWVudHMgRnJlZXplXVxuICAgICAgICBGRUFUVVJFX0JMT0dfRlJFRVpFW0ZlYXR1cmUgQmxvZyBGcmVlemVdXG4gICAgICAgIENPREVfRlJFRVpFW0NvZGUgRnJlZXplXVxuICAgICAgICBET0NTX1BMQUNFSE9MREVSX0ZSRUVaRVtEb2NzIFBsYWNlaG9sZGVyIFBSIERlYWRsaW5lXVxuICAgICAgICBURVNUX0ZSRUVaRVtUZXN0IEZyZWV6ZV1cbiAgICAgICAgRE9DU19SRUFEWV9GT1JfUkVWSUVXW0RvY3MgUFIgUmVhZHkgRm9yIFJldmlldyBEZWFkbGluZV1cbiAgICAgICAgRkVBVFVSRV9CTE9HX1JFQURZX0ZPUl9SRVZJRVdbRmVhdHVyZSBCbG9nIFBSIFJlYWR5IEZvciBSZXZpZXcgRGVhZGxpbmVdXG4gICAgICAgIERPQ1NfRlJFRVpFW0RvY3MgRnJlZXplXVxuICAgICAgICBFTkRbUmVsZWFzZSBEYXldXG5cbiAgICAgICAgU1RBUlQgLS0-IFNUQVJUX0VOSEFOQ0VNRU5UU19UUkFDS0lOR1xuICAgICAgICBTVEFSVF9FTkhBTkNFTUVOVFNfVFJBQ0tJTkcgLS0-IEVOSEFOQ0VNRU5UU19GUkVFWkVcbiAgICAgICAgRU5IQU5DRU1FTlRTX0ZSRUVaRSAtLT4gRkVBVFVSRV9CTE9HX0ZSRUVaRVxuICAgICAgICBGRUFUVVJFX0JMT0dfRlJFRVpFIC0tPiBDT0RFX0ZSRUVaRVxuICAgICAgICBDT0RFX0ZSRUVaRSAtLT4gRE9DU19QTEFDRUhPTERFUl9GUkVFWkVcbiAgICAgICAgRE9DU19QTEFDRUhPTERFUl9GUkVFWkUgLS0-IFRFU1RfRlJFRVpFXG4gICAgICAgIFRFU1RfRlJFRVpFIC0tPiBET0NTX1JFQURZX0ZPUl9SRVZJRVdcbiAgICAgICAgRE9DU19SRUFEWV9GT1JfUkVWSUVXIC0tPiBET0NTX0ZSRUVaRVxuICAgICAgICBET0NTX0ZSRUVaRSAtLT4gRkVBVFVSRV9CTE9HX1JFQURZX0ZPUl9SRVZJRVdcbiAgICAgICAgRkVBVFVSRV9CTE9HX1JFQURZX0ZPUl9SRVZJRVcgLS0-IEVORFxuICAgIGVuZFxuXG4gICAgc3ViZ3JhcGggQmVmb3JlIHRoZSBSZWxlYXNlXG4gICAgICAgIERJU0NVU1NbSW50cm9kdWNlIGFuZCBkaXNjdXNzIHlvdXIgZW5oYW5jZW1lbnRzIHdpdGggc3BvbnNvcmluZyBhbmQgcGFydGljaXBhdGluZyBTSUdzXVxuICAgICAgICBJU1NVRVtDcmVhdGUgb3IgVXBkYXRlIEVuaGFuY2VtZW50IElzc3VlXVxuICAgICAgICBLRVBbQ3JlYXRlIG9yIFVwZGF0ZSBLdWJlcm5ldGVzIEVuaGFuY2VtZW50cyBQcm9wb3NhbF1cblxuICAgICAgICBESVNDVVNTIC0tPiBJU1NVRVxuICAgICAgICBJU1NVRSAtLT4gS0VQXG4gICAgZW5kXG5cbiAgICBzdWJncmFwaCBEdXJpbmcgdGhlIFJlbGVhc2VcbiAgICAgICAgT1BUX0lOW09wdC1pbiB0byB0aGUgUmVsZWFzZV1cbiAgICAgICAgS0VQX0RPTkVbS0VQIEFwcHJvdmVkIGFuZCBNZXJnZWRdXG4gICAgXG4gICAgICAgIENPREVfQ09NUExFVEVbQWxsIGsvayBQUnMgTWVyZ2VkXVxuICAgICAgICBURVNUX0NPTVBMRVRFW0FsbCBUZXN0IFBScyBNZXJnZWRdXG4gICAgICAgIFxuICAgICAgICBGRUFUVVJFX0JMT0dfT1BUX0lOW09wdC1pbiBmb3IgRmVhdHVyZSBCbG9nXVxuICAgICAgICBGRUFUVVJFX0JMT0dfUE9TVF9DT01QTEVURVtDb21wbGV0ZSBGZWF0dXJlIEJsb2ddXG5cbiAgICAgICAgRE9DU19QTEFDRUhPTERFUl9DT01QTEVURVtEb2NzIFBsYWNlaG9sZGVyIFBSIENyZWF0ZWRdXG4gICAgICAgIERPQ1NfUkVBRFlfVE9fUkVWSUVXX0NPTVBMRVRFW0RvY3MgRmluaXNoZWQsIFBSIFJlYWR5IGZvciBSZXZpZXddXG4gICAgICAgIERPQ1NfQ09NUExFVEVbay93ZWJzaXRlIFBSIE1lcmdlZF1cblxuICAgICAgICBLRVAgLS0-IE9QVF9JTlxuICAgICAgICBPUFRfSU4gLS0-IHxXb3JrIE9uIEtFUHwgS0VQX0RPTkVcbiAgICAgICAgXG4gICAgICAgIEtFUF9ET05FIC0tPiB8V29yayBPbiBDb2RlfCBDT0RFX0NPTVBMRVRFXG4gICAgICAgIEtFUF9ET05FIC0tPiB8V29yayBPbiBEcmFmdCBEb2N8IERPQ1NfUExBQ0VIT0xERVJfQ09NUExFVEVcbiAgICAgICAgS0VQX0RPTkUgLS0-IHxXb3JrIE9uIFRlc3R8IFRFU1RfQ09NUExFVEVcblxuICAgICAgICBLRVBfRE9ORSAtLT4gfCBPcHRpb25hbCB8IEZFQVRVUkVfQkxPR19PUFRfSU4gXG4gICAgICAgIEZFQVRVUkVfQkxPR19PUFRfSU4gLS0tLS0tLS0-IHxXb3JrIE9uIEJsb2cgUG9zdHwgRkVBVFVSRV9CTE9HX1BPU1RfQ09NUExFVEVcblxuICAgICAgICBET0NTX1BMQUNFSE9MREVSX0NPTVBMRVRFIC0tPiB8Q29tcGxldGUgRG9jfCBET0NTX1JFQURZX1RPX1JFVklFV19DT01QTEVURVxuICAgICAgICBET0NTX1JFQURZX1RPX1JFVklFV19DT01QTEVURSAtLS0-IHxBZGRyZXNzIEZlZWRiYWNrIEZyb20gUmV2aWV3ZXJzIHwgRE9DU19DT01QTEVURVxuICAgIGVuZFxuXG4gICAgc3ViZ3JhcGggQWZ0ZXIgdGhlIFJlbGVhc2VcbiAgICAgICAgRU5EIC0tPiBGRUFUVVJFX0JMT0dfUFVCTElTSEVEW0ZlYXR1cmUgQmxvZ3MgUHVibGlzaGVkXVxuXG4gICAgZW5kXG5cbiAgICBPUFRfSU4gLS4tIFNUQVJUX0VOSEFOQ0VNRU5UU19UUkFDS0lOR1xuICAgIEtFUF9ET05FIC0uLSBFTkhBTkNFTUVOVFNfRlJFRVpFXG4gICAgQ09ERV9DT01QTEVURSAtLi0gQ09ERV9GUkVFWkVcbiAgICBURVNUX0NPTVBMRVRFIC0uLSBURVNUX0ZSRUVaRVxuICAgIERPQ1NfUExBQ0VIT0xERVJfQ09NUExFVEUgLS4tIERPQ1NfUExBQ0VIT0xERVJfRlJFRVpFXG4gICAgRE9DU19SRUFEWV9UT19SRVZJRVdfQ09NUExFVEUgLS4tIERPQ1NfUkVBRFlfRk9SX1JFVklFV1xuICAgIERPQ1NfQ09NUExFVEUgLS4tIERPQ1NfRlJFRVpFXG4gICAgRkVBVFVSRV9CTE9HX09QVF9JTiAtLi0gRkVBVFVSRV9CTE9HX0ZSRUVaRVxuICAgIEZFQVRVUkVfQkxPR19QT1NUX0NPTVBMRVRFIC0uLSBGRUFUVVJFX0JMT0dfUkVBRFlfRk9SX1JFVklFVyIsIm1lcm1haWQiOiJ7XG4gIFwidGhlbWVcIjogXCJkZWZhdWx0XCJcbn0iLCJ1cGRhdGVFZGl0b3IiOnRydWUsImF1dG9TeW5jIjp0cnVlLCJ1cGRhdGVEaWFncmFtIjp0cnVlfQ) I just added styling & a short description)

#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

Add diagrams to visualize the release team process.

#### Which issue(s) this PR fixes:

Fixes #https://github.com/kubernetes/sig-release/issues/1390

#### Special notes for your reviewer:

/cc @annajung

### CODE REVIEW

File: release-team/README.md [hunk 1]
```diff
@@ -169,3 +170,161 @@ The process for filing an enhancement exception can be found [here][exceptions].
...
+    class START,END plain;
+    class SET,EF k8s;
+    class DPD,DRD,DCF k8s;
+    class FBOD,RBRD,FBRD k8s;
+    class CMT,FDRN,RNC k8s;
+    class TA,EIP,TF k8s;
+    class MONITOR,OPEN,CF,ESCALATE,EAARB,TEST k8s;
+```
+
+The diagram below shows a high level overview of the KEP process during a release cycle.
```
Comment (annajung) [inline release-team/README.md:242]: I would rephrase this to say the relationship between KEP author's work during the release and how it relates to the release deadlines

**Comment by leonardpahlke** [comment]
/hold
I was thinking if I draw up a diagram to visualize also the release cutting process. 

WDYT? @kubernetes/sig-release-leads

**Comment by saschagrunert** [comment]
> /hold I was thinking that I might want to create a diagram to visualize the release cutting process as well (branch manager workflow).
> 
> WDYT? @kubernetes/sig-release-leads

I think that's a great idea!

**Comment by cici37** [comment]
This looks great! Thank you for working on it!

**Comment by leonardpahlke** [comment]
@saschagrunert I have added a rough overview of the release cutting process. Maybe it would be good to deepen it to understand the process steps in more detail.

File: release-engineering/role-handbooks/branch-manager.md [hunk 2]
```diff
@@ -887,3 +889,17 @@ See the branch management process prior to v1.12 when `anago` was still used.
...
+
+# Visual Release Cut Process
+
+The diagram below shows the actions needed to cut a Kubernetes release
+
+```mermaid
+flowchart TD
+    Start --> issue["Create a release cut issue on GitHub"] -->|"update along the way"| issue
+    issue --> thread["Create a Slack Thread in #release-management \n and cc release-managers"] -->|"update along the way"| thread
+    thread --> mock["Mock Release Cut"]
```
Comment (saschagrunert) [inline release-engineering/role-handbooks/branch-manager.md:901]:
```suggestion
    thread --> mock["Mock run (stage and release)"]
```

File: release-engineering/role-handbooks/branch-manager.md [hunk 3]
```diff
@@ -887,3 +889,17 @@ See the branch management process prior to v1.12 when `anago` was still used.
...
 [release-managers]: /release-managers.md#release-managers
 [release-managers-group]: https://groups.google.com/a/kubernetes.io/forum/#!forum/release-managers
+
+# Visual Release Cut Process
+
+The diagram below shows the actions needed to cut a Kubernetes release
+
+```mermaid
+flowchart TD
+    Start --> issue["Create a release cut issue on GitHub"] -->|"update along the way"| issue
```
Comment (saschagrunert) [inline release-engineering/role-handbooks/branch-manager.md:899]: I'd include the ask for Google Build admins availability on the release day.

File: release-engineering/role-handbooks/branch-manager.md [hunk 4]
```diff
@@ -887,3 +889,17 @@ See the branch management process prior to v1.12 when `anago` was still used.
...
+    issue --> thread["Create a Slack Thread in #release-management \n and cc release-managers"] -->|"update along the way"| thread
+    thread --> mock["Mock Release Cut"]
+    mock --> staging["Kick off Release"]
```
Comment (saschagrunert) [inline release-engineering/role-handbooks/branch-manager.md:902]:
```suggestion
    mock --> staging["Nomock stage"]
```

File: release-engineering/role-handbooks/branch-manager.md [hunk 5]
```diff
@@ -887,3 +889,17 @@ See the branch management process prior to v1.12 when `anago` was still used.
...
+    staging --> artifact_promotion["Promote and Sign Artifacts"]
```
Comment (saschagrunert) [inline release-engineering/role-handbooks/branch-manager.md:903]:
```suggestion
    staging --> artifact_promotion["Image promotion"]
```
(they're already signed technically)

File: release-engineering/role-handbooks/branch-manager.md [hunk 6]
```diff
@@ -887,3 +889,17 @@ See the branch management process prior to v1.12 when `anago` was still used.
...
+    artifact_promotion --> release["Release New Kubernetes Version"]
```
Comment (saschagrunert) [inline release-engineering/role-handbooks/branch-manager.md:904]:
```suggestion
    artifact_promotion --> release["Nomock release"]
```

File: release-engineering/role-handbooks/branch-manager.md [hunk 7]
```diff
@@ -887,3 +889,17 @@ See the branch management process prior to v1.12 when `anago` was still used.
...
+```
```
Comment (saschagrunert) [inline release-engineering/role-handbooks/branch-manager.md:905]: - Then we have to get in contact with the Google build admin to cut the debs/rpms
- Then we notify the channel that the release has been cut
- Then we send the announcement via krel
- Then we close the issue if no follow-up work is required

**Review by saschagrunert — APPROVED** [review]
/lgtm
